bundles/org.openhab.binding.mycroft/README.md

```diff
@@ -0,0 +1,161 @@
+
+Possibilies include :
+
+- Press a button in OpenHAB to wake Mycroft without using a wake word.
+- Simulate a voice command to launch a skill, as if you just spoke it
+- Send some text that Mycroft will say (Using its Text To Speach service)
+- Control the music player
+- Control the sound volume of Mycroft
+- React to all the aforementioned events ...
+- ... And send/receive all other kind of messages on the message bus
+
+
+## Supported Things
+
+The only thing managed by this binding is a Mycroft instance.
+
+
+## Discovery
+
+There is no discovery service, as Mycroft doesn't announce itself on the network.
+
+
+## Thing Configuration
+
+The configuration is simple, as you just need to give the IP/hostname of the Mycroft instance accessible on the network.
+The default port is 8181, but you could change it if you want.
+
+```
+Thing mycroft:mycroft:myMycroft "Mycroft A.I." @ "Living Room" [host="192.168.X.X"]
+```
+
+|   property    |              type               |         description                                                     | mandatory |
+|---------------|---------------------------------|-------------------------------------------------------------------------|-----------|
+| host          | IP or string                    | IP address or hostname                                                  |   Yes     |
+| port          | integer                         | Port to reach Mycroft (default 8181)                                    |   No      |
+
+
+## Channels
+
+The Mycroft thing supports the following channels :
+
+
+| channel type id              | Item type | description                                                                                    |
+|------------------------------|-----------|------------------------------------------------------------------------------------------------|
+| listen                       | Switch    | Switch to ON when Mycroft is listening. Can simulate a wake word detection to trigger the STT  |
+| speak                        | String    | The last sentence Mycroft speaks, or ask Mycroft to say something                              |
+| utterance                    | String    | The last utterance Mycroft receive, or ask Mycroft something                                   |
+| player                       | Player    | The music player Mycroft is currently controlling                                              |
+| volume                       | Dimmer    | The volume of the Mycroft speaker. NOT FUNCTIONNAL. [SEE THIS POST TO SEE EVOLUTION](https://community.mycroft.ai/t/openhab-plugin-development-audio-volume-message-types-missing/10576) |
+| volume_mute                  | Switch    | Mute the Mycroft speaker                                                                       |
+| volume_duck                  | Switch    | Duck the volume of the Mycroft speaker                                                         |
```

Comment:
Out of curiosity: What does "duck" mean in this context?

Reply:
In audio engineering, ducking is reducing the level of one audio signal by the presence of another signal.
For mycroft, ducking means reducing volume level temporary (for example, the music volume when mycroft needs to talk, or needs to listen).
Extended to openHAB, we could imagine ducking mycroft volume when a phone is ringing or something like that.